packages/client/playwright.config.ts

```diff
@@ -4,7 +4,7 @@ import { devices } from "@playwright/test"
 const config: PlaywrightTestConfig = {
   testDir: "./e2e",
   /* Maximum time one test can run for. */
-  timeout: 30_000,
+  timeout: 60_000,
```

Comment:
Set temporarily to 60 seconds as 30 is too aggressive.  Permanent timeout changes to be done as part of https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/2288